### PR TITLE
New opcodes

### PR DESCRIPTION
**Pull request by aldoxtor**
I implemented some opcodes.

### CODE REVIEW

**Comment by ShFil119** [comment]
Sorry for complaints( ;) ), but I have a few comments.

You should squash commits with rebase. 
Rand should be replaced by c++11's random. There's also problem of seeding.

It look like there's order "convection":
- Is not implemented?
- unused
- code
Can you move unused earlier?

Last and not least: "ptr != nullptr" is equivalent to "ptr". Additionally "ptr == nullptr" is to "!ptr". ;)

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 1]
```diff
@@ -6370,9 +6349,7 @@ void opcode_0221(const ScriptArguments& args, const ScriptPlayer player, const S
 	@arg arg2 
 */
 void opcode_0222(const ScriptArguments& args, const ScriptPlayer player, const ScriptInt arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x0222);
-	RW_UNUSED(player);
-	RW_UNUSED(arg2);
+	player->getCharacter()->getCurrentState().armour = arg2;
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: This should be health

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 2]
```diff
@@ -6384,9 +6361,7 @@ void opcode_0222(const ScriptArguments& args, const ScriptPlayer player, const S
 	@arg arg2 
 */
 void opcode_0223(const ScriptArguments& args, const ScriptCharacter character, const ScriptInt arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x0223);
-	RW_UNUSED(character);
-	RW_UNUSED(arg2);
+	character->getCurrentState().armour = arg2;
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: Health

**Comment by darkf** [comment]
Any progress on this?

**Comment by husho** [comment]
0098 0099 implemented and merged in #439
00ab 00c5 00c6 0183 0184 01bb 01bc 0222 0223 02a0 02d7 02d8 0369 implemented in #443 
029c 0336 0337 035e 035f 036a 0431 0443 0449 044a 044b still missing

**Comment by darkf** [comment]
@aldoxtor Do you have any intention of finishing this?

**Comment by husho** [comment]
Moved to #481, so I guess this can be closed now.

**Pull request closed without merging**